src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -0,0 +1,30 @@
+package com.courtcanva.ccfranchise.controllers;
+
+import com.courtcanva.ccfranchise.dtos.FranchiseeAndStaffInfoDto;
+import com.courtcanva.ccfranchise.dtos.ResponseDto;
+import com.courtcanva.ccfranchise.services.FranchiseeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
```

Comment:
FYI, @RestController combines @ResponseBody and @Controller under the hood,  which means all return values of methods in the class annotated with this would be wrapped up in Response Body including status and content, you don't need to create other class for wrapping up response value.

Reply:
Thx lao, I didn't understand Responsebody in depth.